R/impute_mean.R

```diff
@@ -113,14 +113,8 @@ trim <- function(x, trim) {
   # Adapted from mean.default
   x <- sort(x, na.last = TRUE)
   na_ind <- is.na(x)
-  if (!is.numeric(trim) || length(trim) != 1L) {
-    cli::cli_abort("{.arg trim} must be numeric of length one.")
-  }
```

Comment:
technically unreachable, since there is a `if (trim == 0) {}` above. but also isn't needed since it should be a rlang type checked earlier anyways